sound/soc/sof/topology.c

```diff
@@ -2641,7 +2632,7 @@ static int sof_link_esai_load(struct snd_soc_component *scomp, int index,
 	}
 
 	/* set config for all DAI's with name matching the link name */
```

Comment:
@ranj063 Can you please remove sdev structure inside sof_link_esai_load and replace it with scomp->dev? As @lgirdwood said we need to find a way to make smaller PR which can be merged and then work on top of them with the rest of the patches.